src/SDKs/ApplicationInsights/Data.ApplicationInsights/generate.cmd

```diff
@@ -0,0 +1,7 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+call %~dp0..\..\..\..\tools\generate.cmd ApplicationInsights/data-plane %*
```

Comment:
Please update this call to correspond with the modified directory structure

Reply:
Fixed